README.md

```diff
@@ -48,6 +48,9 @@ Construct a new Hyperswarm instance.
 * `firewall`: A sync function of the form `remotePublicKey => (true|false)`. If true, the connection will be rejected. Defaults to allowing all connections.
 * `dht`: A DHT instance. Defaults to a new instance.
 
+#### `swarm.connecting`
```

Comment:
Sounds like a boolean but still ok

Reply:
Its like a boolean++. ie 0, swarm is NOT connecting (false). > 0, it is connecting (true) but you can also inspect how many connecting sockets there are.